src/components/List/ListItem.tsx

```diff
@@ -75,7 +87,17 @@ class ListItem extends UIComponent<Extendable<IListItemProps>, any> {
     accessibility: listItemBehavior as Accessibility,
   }
 
-  state: any = {}
+  constructor(props: IListItemProps, context: any) {
```

Comment:
`context` should be omitted as well :) We are not supporting legacy React API

Reply:
done